src/router/views/sensors.vue

```diff
@@ -67,6 +82,36 @@ export default {
         return null
       }
     },
+    filterSensors: function() {
+      let searchValue = this.searchInputTxt.toLowerCase().replace(/\s/g, '')
+
+      // Filtering each item (MAC, name and location) based on the input value
+      let filteredMAC = this.sensors.filter(
```

Comment:
Could try this Array.prototype.some() 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
I am not sure if I can use .some for a single value. It is designed for arrays